openxr/src/foveation_profile_fb.rs

```diff
@@ -0,0 +1,21 @@
+use std::sync::Arc;
+
+use crate::*;
+
+pub struct FoveationProfileFB {
```

Comment:
How would this be used?

Reply:
If we're offering high-level bindings of this extension, it feels odd to have no way to actually *get* a `FoveationProfileFB` value. Also, shouldn't we handle destruction as well?

Reply:
Good points! I added a way to create a profile, and added a `destroy` method. I opted against a destructor because 
1. we need access to the destroy_profile function pointer
2. the struct may have come from a raw handle in which case we dont know if the caller intends to continue using it after this wrapper struct is dropped.
let me know what you think!